### ISSUE TITLE
Uncaught TypeError: editor.getSelection is not a function

### ISSUE THREAD

**Issue by arieljake** [hokaccha/atom-open-github-from-npm#2]
[Enter steps to reproduce below:]
1. selected module name
2. ctrl-alt-n

**Atom Version**: 1.0.19
**System**: Mac OS X 10.11
**Thrown From**: [open-github-from-npm](https://github.com/hokaccha/atom-open-github-from-npm) package, v0.1.0
### Stack Trace

Uncaught TypeError: editor.getSelection is not a function

```
At /Users/arieljake/.atom/packages/open-github-from-npm/lib/open-github-from-npm.coffee:21

TypeError: editor.getSelection is not a function
  at Object.module.exports.open (/Users/arieljake/.atom/packages/open-github-from-npm/lib/open-github-from-npm.coffee:21:19)
  at atom-workspace.subscriptions.add.atom.commands.add.open-github-from-npm:open (/Users/arieljake/.atom/packages/open-github-from-npm/lib/open-github-from-npm.coffee:13:40)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:245:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:524:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:347:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:184:20)

```
### Commands

```
     -3:48 editor:move-to-first-character-of-line (atom-text-editor.editor.is-focused)
     -3:47.9.0 core:move-to-top (atom-text-editor.editor.is-focused)
  2x -3:47.4.0 core:move-down (atom-text-editor.editor.is-focused)
  2x -3:47 core:move-up (atom-text-editor.editor.is-focused)
  2x -3:46.6.0 editor:newline (atom-text-editor.editor.is-focused)
  2x -3:46.3.0 core:move-up (atom-text-editor.editor.is-focused)
  8x -3:42.5.0 core:backspace (atom-text-editor.editor.is-focused)
     -3:39.6.0 core:save (atom-text-editor.editor.is-focused)
     -3:33.1.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -2:25 npm-docs:readme (atom-text-editor.editor.is-focused)
     -1:48.3.0 npm-docs:open (atom-text-editor.editor.is-focused)
     -1:05.9.0 npm-docs:homepage (atom-text-editor.editor.is-focused)
     -0:05.0 open-github-from-npm:open (atom-text-editor.editor.is-focused)
     -0:02.9.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:02.9.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:00.9.0 open-github-from-npm:open (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "themes": [
      "one-dark-ui",
      "atom-light-syntax"
    ],
    "disabledPackages": [
      "language-c",
      "language-clojure",
      "language-csharp",
      "language-go",
      "language-java",
      "language-objective-c",
      "language-perl",
      "language-php",
      "language-python",
      "language-ruby",
      "language-ruby-on-rails",
      "language-toml"
    ],
    "ignoredNames": [
      ".git",
      ".sass-cache",
      ".tmp",
      "bower_components",
      "node_modules"
    ],
    "openEmptyEditorOnStart": false,
    "destroyEmptyPanes": false
  }
}
```
### Installed Packages

``` coffee
# User
atom-beautify, v0.28.14
atom-jade, v0.3.0
build, v0.47.0
build-grunt, v0.2.0
build-npm-apm, v0.2.0
editorconfig, v1.2.0
open-github-from-npm, v0.1.0
project-manager, v2.6.1
tag, v0.3.0

# Dev
No dev packages
```

**Comment by igogo5yo** [comment]
I have the same error!

**Comment by hokaccha** [comment]
fixed at v0.1.1
